apps/bmd/src/pages/PollWorkerScreen.tsx

```diff
@@ -330,29 +466,70 @@ const PollWorkerScreen: React.FC<Props> = ({
             }
           />
         )}
+        {isSavingTally && (
+          <Modal content={<Loading>Saving to pollworker card…</Loading>} />
+        )}
+        {isPrintingCombinedReport && (
+          <Modal content={<Loading>Printing combined report…</Loading>} />
+        )}
+        {isConfirmingCombinedPrint && (
+          <Modal
+            content={
+              <Prose>
+                <p>
+                  Do you want to print the combined results report from the{' '}
+                  {pluralize('machine', machineMetadata.length, true)} (
+                  {machineMetadata.map((m) => m.machineId).join(', ')}) and
+                  clear data from the card?
+                </p>
+              </Prose>
+            }
+            actions={
+              <React.Fragment>
+                <Button primary onPress={requestPrintCombinedMachineReport}>
+                  Print Report
+                </Button>
+                <Button onPress={() => setIsConfirmingCombinedPrint(false)}>
+                  Close
+                </Button>
+              </React.Fragment>
+            }
+          />
+        )}
       </Screen>
       {isPrintMode &&
         reportPurposes.map((reportPurpose) => (
           <React.Fragment key={reportPurpose}>
             <PollsReport
               key={`polls-report-${reportPurpose}`}
               appName={machineConfig.appMode.name}
-              ballotsPrintedCount={ballotsPrintedCount}
+              ballotsPrintedCount={
+                isPrintingCombinedReport
+                  ? combinedBallotsPrinted
+                  : ballotsPrintedCount
+              }
               currentDateTime={currentDateTime}
               election={election}
               isLiveMode={isLiveMode}
-              isPollsOpen={isPollsOpen}
+              isPollsOpen={isPrintingCombinedReport ? false : !isPollsOpen}
```

Comment:
Why did this change to negate `isPollsOpen`? It reads like a mistake, but the logic is a little hard to follow so it's not clear whether it is or not.

Reply:
agreed, I'm confused too.

Reply:
So this report is printed right before the value of isPollsOpen is updated, currently it prints the correct report by having the code in PollsReport print a polls closed report when isPollOpen is true and a polls open report when isPollsOpen is false. I thought this was super confusing so I updated PollsReport to intepret isPollsOpen more "correctly" (true means print polls are open) and instead the confusion is moved into here. 

I agree its still confusing.... but I think this is a bit better then having the PollsReport component do something that seems backwords.

Reply:
I can live with this, but would recommend a comment explaining the meaning of `isPollsOpen` here.